packages/angular-test-app/src/preview-examples/button-loading.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-example',
+  template: `
+    <ix-button id="btn" (click)="loading()" class="m-1" variant="Secondary">
```

Comment:
There should be three variations of the loading button scenario:

1. Button with text (already done)
2. Button with icon + text
3. Icon button